src/System.Private.Xml/src/System/Xml/Core/XmlReaderSettings.cs

```diff
@@ -569,7 +569,7 @@ private void Initialize(XmlResolver resolver)
 
         private static XmlResolver CreateDefaultResolver()
         {
-            return new XmlSystemPathResolver();
+            return new XmlUrlResolver();
```

Comment:
@tarekgh AFAIK UWP test runner on Helix does not allow resolving external paths and that's why these tests were disabled for UAP. Issue tracking this: https://github.com/dotnet/corefx/issues/13121

I did test this change on UAP locally and it threw the expected ArgumentException.